### PR TITLE
added transitive managers operation

### PR DESCRIPTION
**Pull request by Licantrop0**

### CODE REVIEW

File: api-reference/v1.0/api/user-list-manager.md [hunk 1]
```diff
@@ -27,24 +30,44 @@ One of the following permissions is required to call this API. To learn more, in
 <!-- { "blockType": "ignored" } -->
 ```http
 GET /me/manager
+GET /me?$expand=manager($levels=max)
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:33]: The title and file name of this operation is List, but the operation is a GET. Should we update the title/file name?

File: api-reference/v1.0/api/user-list-manager.md [hunk 2]
```diff
@@ -27,24 +30,44 @@ One of the following permissions is required to call this API. To learn more, in
...
 GET /users/{id | userPrincipalName}/manager
+GET /users/{id | userPrincipalName}/?$expand=manager($levels=max)
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:35]: For the expand scenarios, you might want a separate HTTP block that you introduce to explain that that syntax expands the manager's chain.

File: api-reference/v1.0/api/user-list-manager.md [hunk 3]
```diff
@@ -27,24 +30,44 @@ One of the following permissions is required to call this API. To learn more, in
...
 ```
+
 ## Optional query parameters
-This method supports the [OData query parameters](/graph/query-parameters) to help customize the response.
+
+This method supports the [OData query parameters](/graph/query-parameters) to help customize the response.  
+`$expand` manager query requires:
+
+- `?$expand=manager($levels=max)` QueryString parameter
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:43]: This is a little bit confusing; it appears to be saying that the expanding the manager requires the expand manager parameter? Do you mean something like "If your request includes the `$expand` parameter to get the manager's chain, you must also include the following:" Then also show an example of the full $expand parameter syntax.

File: api-reference/v1.0/api/user-list-manager.md [hunk 4]
```diff
@@ -27,24 +30,44 @@ One of the following permissions is required to call this API. To learn more, in
...
 ## Response
 
 If successful, this method returns a `200 OK` response code and a [directoryObject](../resources/directoryobject.md) object in the response body.
-## Example
-##### Request
-The following is an example of the request.
+
+## Examples
+
+##### Get Manager Request
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:68]: When we include multiple examples, we make each example an H3 (###) and we preface with Example 1:, Example 2: and so on. The Request and Response headers are H4s (####).

File: api-reference/v1.0/api/user-list-manager.md [hunk 5]
```diff
@@ -27,24 +30,44 @@ One of the following permissions is required to call this API. To learn more, in
...
+>**Note:** `max` is the only allowed value for `$levels`.  
+> When `$level` parameter is not specified, only the immediate manager is returned.  
+> You can specify `$select` inside `$expand` to select the manager's properties: `$expand=manager($levels=max;$select=id,displayName)`
+
 ## Request headers
+
 | Header       | Value|
 |:-----------|:------|
 | Authorization  | Bearer {token}. Required.  |
+| ConsistencyLevel | eventual. Needed only for `$expand` with `$levels=max` |
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:56]: Revised the description as: 'Eventual. Required when the request includes the `expand` parameter."

File: api-reference/v1.0/api/user-list-manager.md [hunk 6]
```diff
@@ -72,39 +95,91 @@ GET https://graph.microsoft.com/v1.0/users/{id|userPrincipalName}/manager
...
-  "mailNickname": "SaraD",
-  "state": "CA",
-  "streetAddress": "9256 Towne Center Dr., Suite 400",
-  "surname": "Davis",
-  "usageLocation": "US",
   "userPrincipalName": "SaraD@contoso.onmicrosoft.com"
 }
 ```
 
+##### Get Transitive Managers Request
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:121]: For the H3 here, suggest: "Example 2: Get manager chain up to the root level".

File: api-reference/v1.0/api/user-list-manager.md [hunk 7]
```diff
@@ -72,39 +95,91 @@ GET https://graph.microsoft.com/v1.0/users/{id|userPrincipalName}/manager
...
+
+The following is an example of the Get Transitive Managers request.
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "get_transitive_managers"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/me?$expand=manager($levels=max;$select=id,displayName)&$select=id,displayName&$count=true
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:131]: This example doesn't show the ConsistencyLevel header.

File: api-reference/v1.0/api/user-list-manager.md [hunk 8]
```diff
@@ -72,39 +95,91 @@ GET https://graph.microsoft.com/v1.0/users/{id|userPrincipalName}/manager
...
+```
+# [C#](#tab/csharp)
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:133]: Remove the snippet links; for new requests, the snippets will get created via the snippet generation automation. Also remove the HTTP H1.

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please see editorial comments. Also, we'll need to make the same updates to the beta version of this topic.

**Review by davidmu1 — COMMENTED** [review]
I don't have anything to add in addition to what Laura has pointed out.

File: api-reference/v1.0/api/user-list-manager.md [hunk 9]
```diff
@@ -11,8 +11,10 @@ doc_type: apiPageType
 
 Namespace: microsoft.graph
 
-Get user's manager. Returns the user or organizational contact assigned as the user's manager.
+Returns the user or organizational contact assigned as the user's manager. Optionally, you can expand the manager's chain up to the root node.
```
Comment (Lauragra) [inline api-reference/v1.0/api/user-list-manager.md:14]: This PR updates the v1.0 version of the topic. Please apply the same updates to the beta version of the topic as well.

**Review by Lauragra — CHANGES_REQUESTED** [review]
We need to make the same updates to the beta topic.

**Review by Lauragra — APPROVED** [review]

**Comment by Lauragra** [comment]
@Licantrop0 , please address the API Doctor errors:
Error: get_manager:
      Missing properties: response was missing these required properties: aboutMe, accountEnabled, ageGroup, assignedLicenses, assignedPlans, birthday, businessPhones, city, companyName, consentProvidedForMinor, country, createdDateTime, creationType, department, employeeHireDate, employeeId, employeeOrgData, employeeType, faxNumber, givenName, hireDate, identities, imAddresses, interests, isResourceAccount, legalAgeGroupClassification, licenseAssignmentStates, lastPasswordChangeDateTime, mailboxSettings, mailNickname, mobilePhone, mySite, officeLocation, onPremisesDistinguishedName, onPremisesDomainName, onPremisesExtensionAttributes, onPremisesImmutableId, onPremisesLastSyncDateTime, onPremisesProvisioningErrors, onPremisesSamAccountName, onPremisesSecurityIdentifier, onPremisesSyncEnabled, onPremisesUserPrincipalName, otherMails, passwordPolicies, passwordProfile, pastProjects, postalCode, preferredLanguage, preferredName, provisionedPlans, proxyAddresses, responsibilities, schools, showInAddressList, signInSessionsValidFromDateTime, skills, state, streetAddress, surname, usageLocation, userType, drives, inferenceClassification, outlook

  Error: get_transitive_managers:
      Missing properties: response was missing these required properties: aboutMe, accountEnabled, ageGroup, assignedLicenses, assignedPlans, birthday, businessPhones, city, companyName, consentProvidedForMinor, country, createdDateTime, creationType, department, employeeHireDate, employeeId, employeeOrgData, employeeType, faxNumber, givenName, hireDate, identities, imAddresses, interests, isResourceAccount, jobTitle, legalAgeGroupClassification, licenseAssignmentStates, lastPasswordChangeDateTime, mail, mailboxSettings, mailNickname, mobilePhone, mySite, officeLocation, onPremisesDistinguishedName, onPremisesDomainName, onPremisesExtensionAttributes, onPremisesImmutableId, onPremisesLastSyncDateTime, onPremisesProvisioningErrors, onPremisesSamAccountName, onPremisesSecurityIdentifier, onPremisesSyncEnabled, onPremisesUserPrincipalName, otherMails, passwordPolicies, passwordProfile, pastProjects, postalCode, preferredLanguage, preferredName, provisionedPlans, proxyAddresses, responsibilities, schools, showInAddressList, signInSessionsValidFromDateTime, skills, state, streetAddress, surname, usageLocation, userPrincipalName, userType, drives, inferenceClassification, outlook

File: api-reference/beta/api/user-list-manager.md [hunk 10]
```diff
@@ -26,59 +28,137 @@ One of the following permissions is required to call this API. To learn more, in
...
-Here is an example of the response.
-<!-- { "blockType": "ignored" } -->
+
+#### Response
+
+The following is an example of the response.
+>**Note**: The response object shown here might be shortened for readability.
+<!-- {
+  "blockType": "response",
+  "truncated": false,
```
Comment (finsharp) [inline api-reference/beta/api/user-list-manager.md:97]: @Licantrop0 This response is marked as ` "truncated": false` yet from the json object below, its truncated. Was your intention to put a truncated response object?

File: api-reference/beta/api/user-list-manager.md [hunk 11]
```diff
@@ -26,59 +28,137 @@ One of the following permissions is required to call this API. To learn more, in
...
+```
+
+#### Response
+
+The following is an example of the response. Transitive managers are displayed hierarchically.
+
+>**Note**: The response object shown here might be shortened for readability.
+<!-- {
+  "blockType": "response",
+  "truncated": false,
```
Comment (finsharp) [inline api-reference/beta/api/user-list-manager.md:136]: @Licantrop0 Similar to the above comment on response truncation.

**Pull request merged**